WordPress/Classes/ViewRelated/Media/MediaItemViewController.swift

```diff
@@ -146,7 +151,8 @@ class MediaItemViewController: UITableViewController {
         SVProgressHUD.show(withStatus: NSLocalizedString("Deleting...", comment: "Text displayed in HUD while a media item is being deleted."))
 
         let service = MediaService(managedObjectContext: ContextManager.sharedInstance().mainContext)
-        service.delete(media, success: {
+        service.delete(media, success: { [weak self] in
+            WPAppAnalytics.track(.mediaLibraryDeletedItems, withProperties: ["number_photos_deleted": 1], with: self?.media.blog)
```

Comment:
Maybe this should be `"number_of_items_deleted"` since we can also delete videos here.